pkg/nakadi/nakadi.go

```diff
@@ -62,6 +62,90 @@ func (c *Client) UnconsumedEvents(ctx context.Context, subscriptionID string) (i
 	return unconsumedEvents, nil
 }
 
+type SubscriptionFilter struct {
+	SubscriptionID    string
+	OwningApplication string
+	EventTypes        []string
+	ConsumerGroup     string
+}
+
+func (c *Client) subscriptions(ctx context.Context, filter *SubscriptionFilter, href string) ([]string, error) {
```

Comment:
Just a general thought, this raw HTTP stuff doesn't look very nice. I would imagine there are many use cases for doing this kind of a thing with the nakadi API:
1. Have some query parameters.
2. Send a query requests to nakadi endpoint.
3. Process the response

Don't we have some kind of a client library we can use? 🤔

Reply:
Same question about the `stats` endpoint, there's a lot of raw HTTP stuff there.

Reply:
There is https://github.bus.zalan.do/aruha/nakadi-go, but that's a Zalando-internal library, not supported, and (from a short look) only has the "get subscription" without the consumer group parameter.